front-end/src/app/transactions/transaction-group-d/transaction-group-d.component.ts

```diff
@@ -30,10 +30,25 @@ export class TransactionGroupDComponent extends TransactionTypeBaseComponent imp
     'memo_code',
     'memo_text_description',
   ];
+  readOnlyMemo = false;
```

Comment:
i think renaming this to readOnlyMemoCode would be more clear what input it is affecting